FrontDesk/tests/UnitTests/Core/AggregatesEntities/AppointmentTests/Appointment_Constructor.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using FrontDesk.Core.Aggregates;
+using PluralsightDdd.SharedKernel;
+using Xunit;
+
+namespace UnitTests.Core.AggregatesEntities.AppointmentTests
+{
+  public class Appointment_Constructor
+  {
+    [Fact]
+    public void ConstructorCreatedAndIdAutoGenerated()
+    {
+      var scheduleId = Guid.NewGuid();
+      var range = new DateTimeRange(DateTime.Now, DateTime.Now.AddHours(3));
+      var appointment = new Appointment(1, scheduleId, 2, 3, 4, 5, range, "Title Test");
+
+      Assert.Null(appointment.DateTimeConfirmed);
+      Assert.Equal(1, appointment.AppointmentTypeId);
+      Assert.Equal(scheduleId, appointment.ScheduleId);
+      Assert.Equal(2, appointment.ClientId);
+      Assert.Equal(3, appointment.DoctorId);
+      Assert.Equal(4, appointment.PatientId);
+      Assert.Equal(5, appointment.RoomId);
+      Assert.Equal(3*60, appointment.TimeRange.DurationInMinutes());
+      Assert.Equal("Title Test", appointment.Title);
+      Assert.NotEqual(Guid.Empty, appointment.Id);
+    }
```

Comment:
Using different numbers is better, but still need to only use each one once. Need to assign each literal number or string to a variable. Start with a variable in the test, and if you need it in more than one test consider moving it to a class field. Example:

```
int testAppointmentTypeId = 1;

var appointment = new Appointment(testAppointmentId, ...);

Assert.Equal(testAppointmentId, appointment.AppointmentTypeId);
```